deploy/cert.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
Is this required?  Is there any drawback to using the built-in certificate generator?  Fewer moving parts.

Reply:
I think there is no more auto generating certs support now